core/lib/config/Cargo.toml

```diff
@@ -23,6 +23,10 @@ anyhow.workspace = true
 rand.workspace = true
 secrecy.workspace = true
 serde = { workspace = true, features = ["derive"] }
+time = { workspace = true, features = ["serde-human-readable"] }
+strum.workspace = true
+vise.workspace = true
+strum_macros = "0.26.4"
```

Comment:
I believe this one should also be moved to workspace deps

Reply:
Fixed, sorry missed that.